### ISSUE TITLE
[ios/droid][Gallery][ExpandingBottomSheet] Close button blocked by nav-view toggle button

### ISSUE THREAD

**Issue by Xiaoy312** [unoplatform/uno#5148]
## Current behavior
![20210204 mobile ebs closing](https://user-images.githubusercontent.com/2359550/106987744-21574c00-673c-11eb-8294-572260520010.gif)
When attempting to close the EBS, instead of EBS handling the click, the nav-view toggle button from underneath handled the event.

**Comment by Xiaoy312** [comment]
Duplicate of unoplatform/Uno.Gallery#36
